src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -456,6 +458,8 @@ std::string ContextAttribute::render
       std::string effectiveValue      = "";
       bool        valueIsNumberOrBool = false;
 
+      using namespace orion;
+
```

Comment:
Yes, you are right... Not sure why I have ended with a duplicated declaration. I'll check.